schemas/session-replay/mobile/_common-wireframe-update-schema.json

```diff
@@ -30,6 +30,26 @@
       "type": "integer",
       "description": "The height in pixels of the UI element, normalized based on the device pixels per inch density (DPI). Example: if a device has a DPI = 2, the height of all UI elements is divided by 2 to get a normalized height.",
       "readOnly": true
+    },
+    "clipTop": {
```

Comment:
**suggestion**: Why not an object called `clip` with properties `top, right, bottom, left` following what we already did for `border` and `shapeStyle` ? This way we'd reuse a single definition.

Reply:
Hmmm...good suggestion, I am going to try and see on my end how this will end up in our schemas.